cartridge-cli.lua

```diff
@@ -3759,22 +3761,31 @@ local function start_all(args)
     local errors = {}
 
     -- start instances
-    for _, process_args in pairs(processes) do
-        local instance_status_str = string.format('Starting %s', process_args.instance_id)
+    for _, process in pairs(processes) do
```

Comment:
`pairs` doesn't respect of array. Is it important for us to give for user predictable result?

Reply:
Let's pretend you didn't see it